From 13333e5c47d8b9cadb8a1ba881a39c05be186553 Mon Sep 17 00:00:00 2001
From: Pavel Valena <pvalena@redhat.com>
Date: Thu, 6 Mar 2025 14:23:27 +0100
Subject: [PATCH 13/31] fix(rescue): create hmac file for rescue kernel

Resolves: RHEL-59543
---
 install.d/51-dracut-rescue.install | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/install.d/51-dracut-rescue.install b/install.d/51-dracut-rescue.install
index 1be9b882..030f45ba 100755
--- a/install.d/51-dracut-rescue.install
+++ b/install.d/51-dracut-rescue.install
@@ -117,6 +117,17 @@ case "$COMMAND" in
             echo "Can't copy '$KERNEL_IMAGE to '$BOOT_DIR_ABS/$KERNEL'!" >&2
         fi
 
+        HMAC=".${KERNEL}.hmac"
+
+        KERNEL_DIR="${KERNEL_IMAGE%/*}"
+        KERNEL_FILE="${KERNEL_IMAGE##*/}"
+
+        HMAC_FILE="${KERNEL_DIR}/.${KERNEL_FILE}.hmac"
+
+        if ! sed -E "s/([0-9a-f]+)(\s+).*$/\1\2${KERNEL}/" "$HMAC_FILE" > "$BOOT_DIR_ABS/$HMAC"; then
+            echo "Can't create '$BOOT_DIR_ABS/$HMAC' from '$HMAC_FILE'!" >&2
+        fi
+
         if [[ ! -f "$BOOT_DIR_ABS/$INITRD" ]]; then
             # shellcheck disable=SC2046
             dracut -f --no-hostonly --no-uefi \
-- 
2.52.0

